source/features/better-diff-selection.tsx

```diff
@@ -0,0 +1,34 @@
+import './better-diff-selection.css';
+
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+let selectedDiffTableBody: HTMLElement | undefined;
+
+function init(): void {
+	delegate(document.body, '.diff-table tr:not(.js-expandable-line) td:nth-child(even)', 'mousedown', event => {
+		const target = event.target as HTMLElement;
+		selectedDiffTableBody = target.closest('tbody')!;
+		selectedDiffTableBody.dataset.rghSelect = target.closest('td') === target.closest('tr')!.children[1] ? 'left' : 'right';
+	});
+
+	document.body.addEventListener('mouseup', () => {
```

Comment:
Perhaps mouseup is too soon. We probably need to listen to a `selectionchange` event so that we reset only when the selection removed.

This is what happens in Chrome:

![a212](https://user-images.githubusercontent.com/1402241/98169199-740d3780-1eb1-11eb-9a9b-957773018a33.gif)

Is/was this not happening before? Any alternative solutions?

Reply:
> Is/was this not happening before?
No, `mousedown`/`mouseup` always worked in Firefox.

Reply:
> We probably need to listen to a selectionchange event so that we reset only when the selection removed. Any alternative solutions?

I tried `preventDefault()`, to no avail. Using `selectionchange` works fine in both browsers (with a small annoyance in Chromium, I'll make a gif of it)

Reply:
I think we can get away with simply resetting the selection and the previously disabled diff column on every `mousedown` event.